docs/Compile.rst

```diff
@@ -361,9 +361,10 @@ To install Chocolatey and the required dependencies:
 
 * Paste in the Chocolatey install command and hit enter
 * Close this ``cmd.exe`` window and open another Admin ``cmd.exe`` in the same way
-* Run the following command::
+* Run the following commands::
 
-    choco install git cmake strawberryperl -y
+    choco install git cmake.portable strawberryperl -y
+    cpanm XML::LibXML
```

Comment:
Can you explain what this is about?

What's the benefit to installing cmake.portable instead of cmake?

Also the CPAN line is definitely not necessary - the reason to install Strawberry Perl vs any other kind of Perl is that it provides both XML::LibXML and XML::LibXSLT already.  The build will work fine without the extra cpan command.